driver/grpc.go

```diff
@@ -444,8 +444,8 @@ func (g *grpcSearchReader) close() error {
 	return nil
 }
 
-func (c *grpcDriver) CreateApplication(ctx context.Context, name string, description string) (*Application, error) {
-	r, err := c.mgmt.CreateApplication(ctx, &api.CreateApplicationRequest{Name: name, Description: description})
+func (c *grpcDriver) CreateApplication(ctx context.Context, name string, description string, project *string) (*Application, error) {
```

Comment:
Do not use pointers. Just pass the string.
Also, move it to be second parameter after the ctx, so as it has higher scope then name and description.

Reply:
refactored.